governance/polls/Approve SPF ETHAmsterdam Event (MIP55c3-SP3) - March 28, 2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Approve SPF ETHAmsterdam Event (MIP55c3-SP3) - March 28, 2022
+summary: Signal your support or opposition to approving SPF ETHAmsterdam Event (MIP55c3-SP3)
+discussion_link: https://forum.makerdao.com/t/mip55c3-sp3-ethamsterdam-event-spf/13781
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-03-28T16:00:00
+end_date: 2022-03-31T16:00:00
+---
+# Poll: Approve SPF ETHAmsterdam Event (MIP55c3-SP3) - March 28, 2022
+
+The Governance Facilitators have placed this poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP4](https://mips.makerdao.com/mips/details/MIP4). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, March, 28 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
```

Comment:
So we should try to match the general MIP subproposal template here, if possible. So including links to the canonical, latest version, etc. Sentence and/or Paragraph summary, etc.

I think the key is that this is subproposal, and should be treated like one. The only difference is that it's eligible to be resolved through the weekly cycle (which we should also call out somewhere, I think)